jormungandr-lib/src/interfaces/vote/mod.rs

```diff
@@ -330,7 +330,49 @@ impl TallyResult {
 }
 
 #[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
-pub struct EncryptedTally(Vec<u8>);
+pub struct EncryptedTally(#[serde(with = "serde_bytes")] Vec<u8>);
+
+mod serde_hex_bytes {
```

Comment:
I don't get it. The attribute says use `serde_bytes` module, what is `mod serde_hex_bytes` for?

Also, this does not distinguish between human-readable and non-human-readable serde backends. For the latter, the serialization should still use a byte vector. We probably don't use any at the moment, but it will be embarrassing if we do and then find out that it uses a wasteful hex encoding.

Reply:
I did not know it can be discriminated. I'll check about it.